corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowser.java

```diff
@@ -118,8 +128,7 @@ private static void verifyNamespaceAndTablename(String namespace, String tableNa
         log.info("Namespace: {}", namespace);
         log.info("TableName: {}", tableName);
 
-        ISerializer dynamicProtobufSerializer =
-                new DynamicProtobufSerializer(runtime);
+        dynamicProtobufSerializer = new DynamicProtobufSerializer(runtime);
```

Comment:
What is this set in a method and not in the constructor ?

Reply:
good point.  I can add it in the constructor

Reply:
@Maithem , adding it in the constructor seems to be registering >1 serializers with the runtime and causes tests to fail. So I dont have that change in the latest commit.  I will try to debug it tomorrow.

Reply:
DynamicProtobufSerializer is only needed by operations like printTable/dropTable/editRecord.  So only creating and registering it in those operations.

In the meantime, will explore how other operations in the tool can use the DynamicProtobufSerializer as well.